### DISCUSSION TITLE
Limit usage of entry types in matrix field

### DISCUSSION THREAD

**Discussion by kennethdenhollander** [craftcms/cms#15730] [Ideas] [answered]
Currently, when using a matrix field (in our case, as a page builder), you can create an unlimited number of a certain entry type. It would be helpful to add a limit, so a specific entry type within the matrix can only be used 'x' times.

In our setup, we sometimes have an external widget that can only be loaded once. Therefore, we would like to restrict authors to use this entry type only once within the matrix, while still allowing them to reorder the different entry types for flexibility reasons.

**Comment by brandonkelly** [comment, marked as answer]
Thanks for the suggestion! It was previously requested by #8596. You can follow along there for updates.
